docs/tasks/policy.md

```diff
@@ -0,0 +1,216 @@
+# Task: Cohesity Policy Management 
+
+## Table of Contents
+- [Synopsis](#synopsis)
+- [Requirements](#requirements)
+- [Ansible Variables](#Ansible-Variables)
+- [Customize Your Playbooks](#Customize-your-playbooks)
+ - Policy Management
+    - [Create a Protection Policy](#Create-a-Protection-Policy)
+    - [Modify a Protection Policy](#Modify-a-Protection-Policy)
+    - [Delete a Protection Policy](#Delete-a-Protection-Policy)
+- [How the Task Works](#How-the-Task-works)
+
+## Synopsis
+[top](#Cohesity-policy-management)
+
+Use this task to clone a VM using Test/Dev workflow
+
+#### How It Works
+- The tasks collects all the information (required and optional) and starts a clone task (if *state=present*).
```

Comment:
this one talks of clone, please change it to policy for this one and the one below